### PR TITLE
fix:  Pipeline editor cant save init value

### PR DESCRIPTION
**Pull request by Mrxyy**
### What type of PR is this?

/kind bug

### What this PR does / why we need it:
When the pipeline is edited again and no input is made, the initial value cannot be saved
### Which issue(s) this PR fixes:

Fixes ##https://github.com/kubesphere/kubesphere/issues/3950

### Special notes for reviewers:
WeChat063a5f86da96bf4f4b628dcade59f3b7
WeChat68682146e1ee42523c88fc7c7899538e

```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: src/pages/devops/components/Pipeline/StepModals/script.jsx [hunk 1]
```diff
@@ -45,23 +45,25 @@ export default class Shell extends React.Component {
...
   }
 
   handleOk = () => {
+    const { initValue } = this.state
     this.props.onAddStep({
       name: 'script',
       arguments: [
         {
           key: 'scriptBlock',
-          value: { isLiteral: true, value: this.newValue || '' },
```
Comment (harrisonliu5) [inline src/pages/devops/components/Pipeline/StepModals/script.jsx:64]: This is the default data structure, which means that there is no child.

**Review by harrisonliu5 — DISMISSED** [review]

**Comment by LinuxSuRen** [comment]
fix https://github.com/kubesphere/ks-devops/issues/106

**Review by LinuxSuRen — APPROVED** [review]
I test it manually using the following image:

`kubespheredev/ks-console:pr-Mrxyy-2134-dev`

/lgtm

**Pull request merged**